spec/tools/fix_auth/auth_config_model_spec.rb

```diff
@@ -31,13 +22,13 @@
           },
           :root_password           => enc_old,
           :sysprep_password        => enc_old,
-          :sysprep_domain_password => enc_new
+          :sysprep_domain_password => enc_old
         )
       )
     end
 
     it "upgrades request (find with prefix, do not stringify keys)" do
-      subject.fix_passwords(request)
+      subject.fix_passwords(request, options)
```

Comment:
not sure if it matters, but could you either pass in `:legacy_key =>legacy_key` or change the variable name for `options`? (I'd prefer the former)

Reply:
I had it this way in test to mirror the way it is in the code.  That is, in the code, the `options` are passed.

Also, not just legacy_key is passed in some cases.  I was also trying to avoid having `:legacy_key => legacy_key` copied over and over, in case later we needed to add more options.

Since I'm changing every spot anyway, I could do that, but I'm really leaning away from doing that and keeping as is.